magenta-lib/src/test/scala/magenta/input/RiffRaffYamlReaderTest.scala

```diff
@@ -0,0 +1,73 @@
+package magenta.input
+
+import org.scalatest.{FlatSpec, ShouldMatchers}
+import play.api.libs.json.{JsArray, JsNumber, JsString, Json}
+
+class RiffRaffYamlReaderTest extends FlatSpec with ShouldMatchers{
```

Comment:
This is required to narrow down failures if the parser code is broken. Having tests of the logic depend on external functionality isn't ideal, but with this additional coverage problems emerging from that coupling are less severe.